ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -113,6 +114,7 @@ public ForkChoice(
       final MergeTransitionBlockValidator transitionBlockValidator,
       final boolean forkChoiceUpdateHeadOnBlockImportEnabled,
       final boolean forkChoiceProposerBoostUniquenessEnabled,
+      final boolean forkChoiceLateBlockReorgEnabled,
       final MetricsSystem metricsSystem) {
```

Comment:
I understand that this is out of scope for this PR (and should be done in a different PR if ever). But we could have a `ForkChoiceParams` object instead of adding flags to the constructor. Imagine how this is going to look when we have 10 different toggles for the ForkChoice? :)

Reply:
at that point it'll probably annoy me enough to refactor :)